tests/fixtures/issues/issue-436.php

```diff
@@ -4930,59 +4877,6 @@
         'lite' => true,
         'standard' => true,
     ],
-    'issue-436-CR' => [
-        'ua' => 'Mozilla/5.0 (iPhone; CPU iPhone OS 10_10 like Mac OS X) AppleWebKit/600.1.4 (KHTML, like Gecko) Version/8.0 Mobile/12B401 Safari/600.1.4',
```

Comment:
I removed this one as well due to the iOS version number (`10_10`).  This has a special case iOS platform (`iOS_B_8_0` I believe), but I don't know how that came to be (I looked at #436 to see if there was an explanation, but there wasn't one).  Any insight?